src/v/storage/storage_resources.cc

```diff
@@ -166,37 +179,54 @@ storage_resources::get_falloc_step(std::optional<uint64_t> segment_size_hint) {
     return step;
 }
 
-adjustable_allowance::take_result
-storage_resources::offset_translator_take_bytes(int32_t bytes) {
+bool storage_resources::offset_translator_take_bytes(
+  int32_t bytes, ssx::semaphore_units& units) {
     vlog(
       stlog.trace,
       "offset_translator_take_bytes {} (current {})",
+      units.count(),
```

Comment:
Late to this party, but I count only two `{}` but three parameters now?

Reply:
nice catch. fmt seems to barf and not give a partial rendering, so these are always good finds.

Reply:
@dotnwat - yeah fmt throws an exception, but at least it doesn't propagate because of the way the logger works (seems designed to prevent this: which is really nice since it avoids the time-bombs like "turn on debug logging and your server crashes because a debug format string was mal-formed").

I guess we aren't using compile-time format string checking in fmt? Perhaps that's best for compile times...

Reply:
huh. i thought some of that was on by default when the compiler supported consteval/c++20 and fmt >= v8 was used.

Reply:
@dotnwat indeed and I talked to @BenPope who said it was on by default. Maybe something about `vlog` disables it. If I had time I'd look into it further, but I don't...